### ISSUE TITLE
dcm2nii interface fails

### ISSUE THREAD

**Issue by hanke** [nipy/nipype#1269]
Just for the record at this point:

````
151109-08:01:50,23 workflow INFO:
         Executing node convert in dir: /tmp/heudiconvtmpmKyyo3/convert
151109-08:01:50,114 workflow INFO:
         Running: dcm2nii -a y -c y -b config.ini -v y -d y -e y -g y -i n -n y -o /tmp/heudiconvtmpmKyyo3/convert -p y -x n -f n /tmp/heudiconvtmpmKyyo3/convert/MR.1.3.46.670589.11.38317.5.0.4512.2014061911091475808
Traceback (most recent call last):
  File "/home/data/psyinf/forrest_gump/satratest/git/bin/heudiconv", line 509, in <module>
    with_prov=args.with_prov)
  File "/home/data/psyinf/forrest_gump/satratest/git/bin/heudiconv", line 447, in convert_dicoms
    with_prov=with_prov)
  File "/home/data/psyinf/forrest_gump/satratest/git/bin/heudiconv", line 300, in convert
    res = convertnode.run()
  File "/home/mih/nipype/nipype/pipeline/engine.py", line 1428, in run
    self._run_interface()
  File "/home/mih/nipype/nipype/pipeline/engine.py", line 1538, in _run_interface
    self._result = self._run_command(execute)
  File "/home/mih/nipype/nipype/pipeline/engine.py", line 1664, in _run_command
    result = self._interface.run()
  File "/home/mih/nipype/nipype/interfaces/base.py", line 1034, in run
    outputs = self.aggregate_outputs(runtime)
  File "/home/mih/nipype/nipype/interfaces/base.py", line 1128, in aggregate_outputs
    raise FileNotFoundError(msg)
nipype.utils.filemanip.FileNotFoundError: File/Directory '['/tmp/heudiconvtmpmKyyo3/convert/20140619_104313WIPsT1W3DTFETR2300TI90007isoFSSENSEs301a1003.nii', '/tmp/heudiconvtmpmKyyo3/convert/20140619_104313WIPsT1W3DTFETR2300TI90007isoFSSENSEs301a1003.nii.gz']' not found for Dcm2nii output 'converted_files'.
Interface Dcm2nii failed to run. 
````

This first (uncompressed) file in this list is indeed to there, but the second one (nii.gz) is the same image and present.

**Comment by hanke** [comment]
This problem is a result of the "fix" for #1267. Output files a double-detected when compressed.

**Comment by hanke** [comment]
What about patching the code again to remove a previously detected .nii file form the outputs if a subsequence 'GZip...' lines references the same file (minus the .gz extension).

However, I have no idea how many other implicit assumptions that would break.
